sql/hive/src/test/scala/org/apache/spark/sql/hive/ErrorPositionSuite.scala

```diff
@@ -23,13 +23,18 @@ import org.scalatest.BeforeAndAfterEach
 
 import org.apache.spark.sql.{AnalysisException, QueryTest}
 import org.apache.spark.sql.catalyst.util.quietly
-import org.apache.spark.sql.hive.test.TestHiveSingleton
+import org.apache.spark.sql.hive.test.{TestHive, TestHiveSingleton}
+import org.apache.spark.sql.internal.SQLConf
 
 class ErrorPositionSuite extends QueryTest with TestHiveSingleton with BeforeAndAfterEach {
   import spark.implicits._
 
+  private  val originalCreateHiveTable = TestHive.conf.createHiveTableByDefaultEnabled
+
+
   override protected def beforeEach(): Unit = {
     super.beforeEach()
+    TestHive.conf.setConf(SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED, true)
```

Comment:
This test suite seems to test SELECT queries mostly. Where do we create tables?

Reply:
Also, it's created in `TestHive`.